docs/pages/blog/mui-x-v7.md

```diff
@@ -0,0 +1,361 @@
+---
+title: Introducing MUI X v7.0.0
+description: Check out what's new in the new major version of the advanced components.
+date: 2024-03-25T08:00:00.000Z
+authors: ['josefreitas']
+tags: ['MUI X', 'Product', 'News']
+card: true
+---
+
+<div style="max-width: 692px; width: 100%; height: 230px; overflow: hidden; margin-bottom: 16px;">
+  <a href="https://github.com/mui/mui-x/releases/tag/v7.0.0">
+    <img src="/static/blog/mui-x-v7/intro.jpg" alt="MUI X v7 - major stable release" width="1200" height="400" style="width: 100%; height: 100%; object-fit: cover; object-position: center;" />
+  </a>
+</div>
+
+After a few months in pre-release, we're thrilled to announce the first stable release of [MUI X v7](https://github.com/mui/mui-x/releases/tag/v7.0.0), packed with new components, exciting features, improved usability and developer experience.
```

Comment:
```suggestion
After a few months in pre-release, we're thrilled to announce the stable release of [MUI X v7](https://github.com/mui/mui-x/releases/tag/v7.0.0), packed with new components, exciting features, improved usability, and developer experience.
```

Would we have a _second_ stable release? I guess everything post-stable is stable, right? Maybe we can ditch that